### ISSUE TITLE
access_token via home.nest.com/session ?

### ISSUE THREAD

**Issue by aschmidt74** [foxthefox/ioBroker.nest#110]
Hej!
vorab; Bin mir nicht sicher ob das dzt. überhaupt funktioniert.

Habe mehrere `nest protect` geräte und mir ein `access_token` auf der `home.nest.com/session` seite (eingelogt) geholt.
Das ist auch das einzige was ich in den `Instance settings` eingetragen habe.
jedoch ... die `nest.0` instance startet nicht.

logs unspezifisch. ("`error	uncaught exception: Cannot read property 'error' of undefined`")

**Comment by foxthefox** [comment]
Wie kann man denn in der jetzigen Zeit ein Zugang bei Nest Entwicklung bekommen?
Den bräuchte ich mal wieder um überhaupt den Adapter wieder zum Testen ans Laufen zu bringen.
Grundsätzlich wäre ein Auszug aus dem Log schon hilfreich, da steht meist in der Nähe auch wo der Fehler aufgetreten ist, bzw. bei welchem Aufruf.
Das würde mir weiterhelfen.

**Comment by aschmidt74** [comment]
naja, ich hab ja auch keinen dev-account, sondern bin als normaler user auf `home.nest.com` eingelogt und bin dann auf `/session` gegangen. Ein `access_token` gibts offenbar auch so. Aber ich habe eben keine ahnung, ob das funktionieren kann?!

nach dem adapter-start schaut das .log so aus:
![NEST0](https://user-images.githubusercontent.com/45806825/217005116-d142ad8b-54bd-43b1-be14-9e70b196c916.PNG)

**Comment by foxthefox** [comment]
OK der screenshot hilft erstmal weiter, danke

**Comment by foxthefox** [comment]
ich habe auf github 1.0.1 erstellt, es sollten die Fehler behoben sein
Ob es nun das EInloggen ermöglicht -> ich bin gespannt

**Comment by aschmidt74** [comment]
hmmm, ist anders jetzt:
![NEST0a](https://user-images.githubusercontent.com/45806825/218337604-cb30dee7-6cd3-4860-a716-ae608f6f2abb.PNG)

**Comment by foxthefox** [comment]
yep, der Fehler im code tritt nicht mehr auf, aber der stream bringt error zurück, den hab ich leider nicht in lesbares umgewandelt. hab ich jetzt ergänzt, müsstest nochmal ais github installieren

**Comment by aschmidt74** [comment]
habe mir noch ein neues `access_token` besorgt, und die session im browser offen gelassen,
aber ein normaler useraccount taugt wohl doch nicht für die API:
`An unknown error occurred: {"type":"error","status":401,"message":"Unauthorized"}`

**Comment by foxthefox** [comment]
Adapter läuft, aber der Zugang ist versperrt. 
Wär ja zu einfach, wenn das jetzt gehen würde.
Irgendwie sehr bescheidener Zustand, bei Nest geht nichts mehr über die Entwicklerschnittstelle und Google hat nicht alle Geräte in der API dabei..
